### PR TITLE
Bitrunning: Avatars get silly hacker names

### PR DESCRIPTION
**Pull request by jlsnow301**
## About The Pull Request
Bit avatars now get corny names while spawning in to the virtual domain. You can change your alias in prefs or have it randomized for s0meth1ng 1337.

Added sechud icons (and thereby orbit ui icons) for bit avatars since "Cyb3rHaxx0r" might be confusing to find in the living players section.
## Why It's Good For The Game
This was done as a request and after discussion in the code channel. 
A little bit of character persistence across simulations.
## Changelog
:cl:
add: Bitrunning: You can now choose your hacker alias in prefs.
add: Bit avatars get orbit icons.
/:cl:

### CODE REVIEW

**Comment by DaydreamIQ** [comment]
Requesting to add BlackWillow69 because funny Critical Role reference

File: strings/names/hackers.txt [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+KatanaKid99
+xXDarkL0rd69Xx
+H4xx0rNinja
+CyberNinja1337
+Ub3r1337Sk1llz
+PhrEaK420
+IRCWizard69
+EliteWarezKing
+Y2KPwn3r
+BBSPh34r
```
Comment (lordpidey) [inline strings/names/hackers.txt:41]: Let's add a few more.

`2038Pr0blem`, `Sp4ceW1nd`, `B0bbyT4bles`, `C0deInjector`

Comment (jlsnow301) [reply strings/names/hackers.txt:41]: I'm not familiar with these

**Comment by jlsnow301** [comment]
Just gonna note that this PR opens the possibility that people name their hacker alias with their character name. While this is doable, you're ultra lame for it

File: strings/names/hackers.txt (hunk 1 above)
Comment (Loafin34) [reply strings/names/hackers.txt:41]: let's add xXR0BU5T_SP355M4NXx for the sake of our tiders..

Comment (GPeckman) [reply strings/names/hackers.txt:41]: The first one is a reference to the year 2038 problem, when Unix time will hit the 32 bit integer limit and overflow. The third one is a reference to this XKCD comic: https://xkcd.com/327/

Not sure about the other two, they might just be generic hacker names.

Comment (jlsnow301) [reply strings/names/hackers.txt:41]: Hoping you guys know you can type in your own creation in the character setup screen

Comment (lordpidey) [reply strings/names/hackers.txt:41]: Space Wind is the in universe equivalent of mountain dew.

Code injection is one of the most infamous types of hacking.

**Pull request merged**